addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -3387,7 +3388,7 @@ export class Wysiwyg extends Component {
                 altData[size] = {
                     'image/jpeg': canvas.toDataURL('image/jpeg', 0.75).split(',')[1],
                 };
-                if (size !== originalSize) {
+                if (size !== originalSize && canExportCanvasAsWebp()) {
                     altData[size]['image/webp'] = canvas.toDataURL('image/webp', 0.75).split(',')[1];
```

Comment:
Here, we already have a jpeg alternative above so we can skip generating more altData if `!canExportCanvasAsWebp()`.